components/FullwidthSlider/src/stories/bem.stories.mdx

```diff
@@ -108,6 +108,107 @@ import "../scripts/story.js";
   {/* eslint-enable react/no-unknown-property */}
 </Canvas>
 
+### Portrait Image
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Portrait Image Slider">
+    <Notification />
+    <div class="denhaag-fullwidth-slider">
+      <div class="denhaag-fullwidth-slider__top">
+        <h2 class="denhaag-fullwidth-slider__top-title">Slider title here</h2>
+        <div class="denhaag-fullwidth-slider__pagination"></div>
+        <div class="denhaag-fullwidth-slider__controls">
+          <button
+            aria-label="Vorige carousel afbeelding"
```

Comment:
Als dit maar in gutenberg in het engels is de aria-label + vertaalbaar.

Reply:
Yep!